ui/components/app/confirm/mmi-signature-section/mmi-signature-section.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { memo, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+
+import { MESSAGE_TYPE } from '../../../../../shared/constants/app';
+import {
+  BackgroundColor,
+  BlockSize,
+  Display,
+  IconColor,
+  TextColor,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import {
+  getAccountByAddress,
+  shortenAddress,
+} from '../../../../helpers/utils/util';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  accountsWithSendEtherInfoSelector,
+  currentConfirmationSelector,
+  getSelectedInternalAccount,
+} from '../../../../selectors';
+import { Box, Icon, IconName, Text } from '../../../component-library';
+
+const MMISignatureMismatchBanner: React.FC = memo(() => {
```

Comment:
Seems to be a duplicate and not being used anywhere... Is there a reason for adding this?

Reply:
Oh that is merge issue, let me delete this.